server/bleep/src/webserver/answer.rs

```diff
@@ -95,36 +94,21 @@ const SNIPPET_COUNT: usize = 13;
 pub(super) async fn handle(
     Query(params): Query<Params>,
     Extension(app): Extension<Application>,
-) -> Result<impl IntoResponse, (StatusCode, Json<super::Response<'static>>)> {
-    let semantic = app.semantic.clone().ok_or_else(|| {
-        (
-            StatusCode::INTERNAL_SERVER_ERROR,
-            super::error(ErrorKind::Configuration, "Qdrant not configured"),
-        )
-    })?;
-
-    let query = parser::parse_nl(&params.q).map_err(|e| {
-        (
-            StatusCode::BAD_REQUEST,
-            super::error(ErrorKind::User, e.to_string()),
-        )
-    })?;
-    let target = query.target().ok_or_else(|| {
-        (
-            StatusCode::BAD_REQUEST,
-            super::error(ErrorKind::User, "missing search target".to_owned()),
-        )
-    })?;
+) -> super::Result<impl IntoResponse> {
```

Comment:
This may look better included in `prelude` instead of relying on typing out `super::Result` at every use?